tic-tac-toe/src/ReactHook/index.js

```diff
@@ -0,0 +1,120 @@
+/* import React, { useState } from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+
+
+function Square(props) {
+  return (
+    <button className="square" onClick={props.onClick}>
+      {props.value}
+    </button>
+  );
+}
+
+function Board(props) {
+  const renderSquare = (i) => {
+    return <Square value={props.squares[i]} onClick={() => props.onClick(i)} />;
+  };
+  return (
+    <div>
+      <div className="board-row">
+        {renderSquare(0)}
+        {renderSquare(1)}
+        {renderSquare(2)}
+      </div>
+      <div className="board-row">
+        {renderSquare(3)}
+        {renderSquare(4)}
+        {renderSquare(5)}
+      </div>
+      <div className="board-row">
+        {renderSquare(6)}
+        {renderSquare(7)}
+        {renderSquare(8)}
+      </div>
+    </div>
```

Comment:
Do you still need this file?

Reply:
The files in ReactHook were to keep documentation of how the react work with hooks looked like before converting to redux.